prov/verbs/src/ep_rdm/verbs_av_ep_rdm.c

```diff
@@ -50,14 +50,14 @@ fi_ibv_rdm_start_connection(struct fi_ibv_rdm_ep *ep,
 	struct rdma_cm_id *id = NULL;
 	assert(ep->domain->rdm_cm->listener);
 
-	if (conn->state != FI_VERBS_CONN_ALLOCATED) {
+	if (conn->state != FI_VERBS_CONN_ALLOCATED)
 		return FI_SUCCESS;
-	}
 
 	if (ep->is_closing) {
-		VERBS_INFO(FI_LOG_AV, "Attempt to start connection with addr %s:%u when ep is closing\n",
-			inet_ntoa(conn->addr.sin_addr),
-			ntohs(conn->addr.sin_port));
+		VERBS_INFO(FI_LOG_AV, "Attempt to start connection with addr "
```

Comment:
Linux coding style (which applies here) says:

    However, never break user-visible strings such as printk messages, because that breaks the ability to grep for them.

Reply:
@fzago-cray Sure, agree with that . But I break the part of this string that is before `%s:%u`.
From my experience I'll grep for the string "Attempt to start connection with addr". Do you agree with that?

Reply:
@fzago-cray Thanks, I've updated the code

Reply:
Btw, that coding style guideline is useless when the string contains stuff like %s:%u.  No one will grep for that.